AttitudeManager/Src/AM.cpp

```diff
@@ -49,7 +62,7 @@ void AttitudeManager::runControlLoopIteration() {
     AttitudeManagerInput control_inputs = getControlInputs();
 
     // Run Control Algorithms
-    AttitudeManagerInput motor_outputs = controlAlgorithm_->run(control_inputs);
+    AttitudeManagerInput motor_outputs = controlAlgorithm_->run(control_inputs, axisPIDs);
```

Comment:
above comment(s) - PID owned by class and we shouldn't need to pass them. 

TLDR: the function that calls `<flightmode>::run` doesn't know or care what PID's exist and how they need to be managed. It just expects the flightmode to update and give motor outputs when `run` is called.